sql/wsrep_mysqld.cc

```diff
@@ -1096,9 +1096,12 @@ int wsrep_init() {
 
     char ssl_opts[4096];
     server_main_callback.populate_wsrep_ssl_options(ssl_opts, sizeof(ssl_opts));
+#pragma GCC diagnostic push
```

Comment:
This should be MY_COMPILER_DIAGNOSTIC_PUSH (and the similar macros for the other changes too)

Reply:
done